.jshintrc

```diff
@@ -0,0 +1,16 @@
+{
+    "node": true
```

Comment:
Should't we use `browser: true` instead and add `define`, `module` and `require` to `globals`?